src/logging/Manager.h

```diff
@@ -287,6 +288,20 @@ class Manager : public plugin::ComponentManager<Component> {
      */
     bool WriteFromRemote(EnumVal* id, EnumVal* writer, const std::string& path, detail::LogRecord&& rec);
 
+    /**
+     * Writes out a batch of log entries received from remote nodes.
+
```

Comment:
Maybe clarify how this relates to `WriteFromRemote` since their comments sound so similar ("writes out log entries" vs "writes out a batch of log entries"). Unlike with `DoPublishlogWrites` there's an ambiguity with "write" and "writes" working as a verb and/or a noun. Maybe `WriteBatchFromRemote()`? (Definitely minor.)

Btw did the formatter put these empty lines here or did you do this? :-) It's a bit unsual.

Reply:
I like `WriteBatchFromRemote()` and added a bit of text to contrast it with `WriteFromRemote()`.

> Btw did the formatter put these empty lines here or did you do this? :-) It's a bit unsual.

Thanks! Fixed.